libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -378,6 +378,16 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("IO_DSHOT", 28, AP_BoardConfig, state.io_dshot, 0),
 #endif
+
+#if AP_CRASHDUMP_ACK_ENABLED
+    // @Param: CRSHDMP_ACK
+    // @DisplayName: CrashDump Acknowledgement Required
+    // @Description: Must have value "1" if a crashdump is present on the system, or prearm failure will be raised.  Do not set this parameter unless the risks of doing so are fully understood.  The presence of a crash dump means that the firmware currently installed has crashed with a hardfault or watchdog. The crash dump file gives diagnostic information which can help in finding the issue. Please check the ArduPilot documentation for more details.
+    // @Values: 0:Crash dump NOT acked,1:Crash dump acked
```

Comment:
maybe change this to, "0:Prearm Crash Dump check active, 1:Crash Dump Pre-arm deactivated"

Reply:
Did this